src/test/java/com/wikia/webdriver/elements/mercury/Category.java

```diff
@@ -0,0 +1,58 @@
+package com.wikia.webdriver.elements.mercury;
```

Comment:
isn't that wikiamobile skin ? then correct pachage would be elements.wikiamobile